app/instance-initializers/head-browser.js

```diff
@@ -1,27 +1,12 @@
-import ENV from '../config/environment';
-
 export function initialize() {
-  if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
-
-  // clear fast booted head (if any)
-  let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
-  let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
-  if (startMeta && endMeta) {
-    let el = startMeta.nextSibling
-    while(el && el !== endMeta) {
-      document.head.removeChild(el);
-      el = startMeta.nextSibling;
-    }
-    document.head.removeChild(startMeta);
-    document.head.removeChild(endMeta);
-  }
+  // do nothing!
```

Comment:
Can you move this comment directly down into the `initialize` method below, and remove this extra function?